operations/reservations.md

```diff
@@ -693,8 +695,10 @@ Updates information about the specified reservation. Note that if any of the fie
 | `AccessToken` | string | required | Access token of the client application. |
 | `Client` | string | required | Name and version of the client application. |
 | `Reason` | string | optional | Reason for updating the reservation. Required when updating the price of the reservation. |
-| `CheckOverbooking` | bool | optional | Whether reservation overbooking is checked. Default value is `true`.  |
-| `CheckRateApplicability ` | bool | optional | Whether the rate applicability check is checked. Default value is `true`.  |
+| `CheckOverbooking` | bool | optional | Whether reservation overbooking is checked. Default value is `true`. |
+| `CheckRateApplicability` | bool | optional | Whether the rate applicability check is checked. Default value is `true`. |
+| `Reprice` | bool | optional | Whether the price should be updated according to updates values. Default value is `true`. |
```

Comment:
I woudnt be sure the reprice parametr is for this. I think its for different usecase. 
I remember it that it used to be used for something else

Reply:
It does the tricks that i load the prices from the rates so if you add new night and dont specify price it will take price from rate adjustments which belongs there IMHO